### DISCUSSION TITLE
error: git-remote-https died of signal 6

### DISCUSSION THREAD

**Discussion by anfoss** [Homebrew/discussions#2542] [Getting started]
Hi,

I am trying to install homebrew on a MacOS 10.15.7 using 

`/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"`

However it fails with

`dyld: Library not loaded: @rpath/libcurl.4.dylib
  Referenced from: /opt/anaconda3/libexec/git-core/git-remote-https
  Reason: Incompatible library version: git-remote-https requires version 12.0.0 or later, but libcurl.4.dylib provides version 11.0.0
error: git-remote-https died of signal 6
Failed during: git fetch --force origin
`

I am not too sure about what is the solution, as in should I downgrade git install homebrew and re-update git or build brew from source after manually dl the repo?

**Comment by carlocab** [comment]
Your Anaconda-installed `git` is broken. Do
```
PATH="/usr/bin:$PATH" /bin/bash -c "$(/usr/bin/curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
```

Alternatively, remove Anaconda's `bin` directory from your `PATH` and try again.
